transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannel.java

```diff
@@ -577,6 +577,7 @@ void epollInReady() {
 
                 int size = readBuf.size();
                 for (int i = 0; i < size; i ++) {
+                    readPending = false;
```

Comment:
should we better do an if check before enter the for loop and also only "update" the field one time.